pyproject.toml

```
@@ -1,3 +1,8 @@
+# Define the build system requirements for the project.
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
 [project]
 name = "ccn-template"
 version = "0.0.1"
```

```suggestion
dynamic = ["version"]
```

This is necessary for `setuptools-scm` to automatically set version number, see [docs](https://github.com/pypa/setuptools_scm#pyprojecttoml-usage)